### ISSUE TITLE
Case sensitivity on embedded images?

### ISSUE THREAD

**Issue by RussellAndrewEdson** [ajayhemanth/Machine-Learning-Workshop#10]
Might be causing some issues with the display of rendered images in the notebooks. e.g. Here's one that looks broken because it's looking for a '.png' image (lowercase) rather than a '.PNG' image (uppercase). So you'd get different results depending on OS used (e.g. Windows doesn't care about case, but Linux does for instance, and it's probably also why the images don't render in the GitHub notebook preview).

**e.g. Broken image on Linux (Ubuntu 20.04, Docker containerized):**
![img1](https://user-images.githubusercontent.com/8025610/137110029-772ee280-ee68-4dd5-9e07-d3b511790582.png)

**Raw markdown:**
![img2](https://user-images.githubusercontent.com/8025610/137110025-f289cbc5-afdf-48d7-afee-0a735754c444.png)

**The actual image in the project directory is a '.PNG' image (i.e. uppercase for the file extension):**
![img3](https://user-images.githubusercontent.com/8025610/137110019-55bd6903-3f35-4386-b732-2f1bfb8bd876.png)

**Changing the Markdown...**
![img4](https://user-images.githubusercontent.com/8025610/137110015-2768d24f-a226-455d-ab27-3f391e56e77d.png)

**...fixes the display.**
![img5](https://user-images.githubusercontent.com/8025610/137110007-4089d914-ae19-412c-add0-a36dba9fac0f.png)

Two ways you could fix this. Going through and making all of the case-sensitive changes to the Markdown is the first, easy way, but perhaps wiser (albeit slightly more effort) would be to have consistent case (e.g. lowercase?) across all of the image extensions anyway.

**Comment by RussellAndrewEdson** [comment]
No worries, happy to help. 

Although it looks like something's odd now with the Activity A notebook file size being x10000 times larger than expected (hopefully it's just caching the R plots and tables?)
![large file size](https://user-images.githubusercontent.com/8025610/137228840-ec01e75b-91a8-405e-853e-66da99ede772.png)

**Comment by ajayhemanth** [comment]
Activity-A was already big  because it was stored with the R output

**Comment by RussellAndrewEdson** [comment]
Indeed, most of the 18MB seems to be mainly in that rotatable figure for the gradient descent. Clearing that one puts the file size back at 20KB (and then an extra 1MB or so for the keras output).
